### DISCUSSION TITLE
Update to V4.11

### DISCUSSION THREAD

**Discussion by matthewdelapole** [microsoft-search/pnp-modern-search#3687] [Q&A] [answered]
Hello,

Currently running v4.10.1.0 but need a little help in updating to v4.11 as this fixes a bug we are experiencing. I thought the app would auto-update to the latest version, is this not the case?

To update , is it as simple as downloading the new package and overwriting the new one in SPO Admin center?

Thanks!
Matthew

**Comment by kasperbolarsen** [comment, marked as answer]
Yes sir :-)

**Reply by matthewdelapole** [reply]
Great, thanks!

**Reply by matthewdelapole** [reply]
@kasperbolarsen - do you know if there is any configuration required to enable the following with the new version? Or should the test update to a brighter colour automatically? https://github.com/microsoft-search/pnp-modern-search/pull/3547

**Reply by wobba** [reply]
For this one it should be automatic.
